### PR TITLE
DEVDOCS-3496: html to md

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-3496](https://jira.bigcommerce.com/browse/DEVDOCS-3496)

## What changed?
Changed bold syntax from HTML to MD

### CODE REVIEW

File: docs/legacy/blueprint-themes/blueprint-email-templates.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Sent to the recipient of a gift certificate.
 | `%%LNG_GiftCertificateEmailInstructions%%` | "For instructions on how to redeem your gift certificate please <a href='%s/giftcertificates.php?action=redeem'>click here</a>." |
 | `%%LNG_GiftCertificateEmailAttached%%` | "Your gift certificate is attached to this email." |
 | `%%LNG_GiftCertificateEmailWarning%%` | "Please download or print a copy of your gift certificate for safe keeping as gift certificates are non-transferable." |
-| `%%GLOBAL_EmailFooter%%` | "<b>%s</b><br><a href="%s.mybigcommerce.com/">http://%s.mybigcommerce.com/</a><br>%s is powered by BigCommerce. <a href="bigcommerce.com">Launch your own store for free</a> with BigCommerce." |
+| `%%GLOBAL_EmailFooter%%` | "**%s**<br><a href="%s.mybigcommerce.com/">http://%s.mybigcommerce.com/</a><br>%s is powered by BigCommerce. <a href="bigcommerce.com">Launch your own store for free</a> with BigCommerce." |
```
Comment (bc-andreadao) [inline docs/legacy/blueprint-themes/blueprint-email-templates.md:23]: Not sure if the %s is supposed to show in the preview? It's like this throughout the document

Comment (slsriehl) [reply docs/legacy/blueprint-themes/blueprint-email-templates.md:23]: yes, %s is supposed to show. That's the character that represents the value of the store subdomain. Let's not mess with any of the html in this email templating table.

File: docs/legacy/blueprint-themes/lng-variables.md [hunk 2]
```diff
@@ -206,8 +206,8 @@ To make it easier for store owners who might wish to translate, or otherwise mod
 | `%%LNG_CouponCode%%` | "Coupon Code" |
 | `%%LNG_CouponCodeIntro%%` | "If you have a coupon code, enter it in the box below and click 'Go'." |
 | `%%LNG_CreateAccount%%` | "Create Account" |
-| `%%LNG_CreateAccountIntro1%%` | <b>Checkout:</b> "Fill in the fields below to create a "<br><b>Express Checkout:</b> "Checkout faster" |
-| `%%LNG_CreateAccountIntro2%%` | <b>Checkout:</b> "account. You'll be able to shop and check out faster on your next visit, check your order status online and optionally receive updates on special offers and events."<br><b>Express Checkout:</b> "Save multiple addresses in your address book" |
+| `%%LNG_CreateAccountIntro1%%` | **Checkout:** "Fill in the fields below to create a "<br>**Express Checkout:** "Checkout faster" |
```
Comment (slsriehl) [inline docs/legacy/blueprint-themes/lng-variables.md:209]:
```suggestion
| `%%LNG_CreateAccountIntro1%%` | **Checkout:** "Fill in the fields below to create an "<br>**Express Checkout:** "Checkout faster" |

```

File: docs/legacy/blueprint-themes/lng-variables.md [hunk 3]
```diff
@@ -206,8 +206,8 @@ To make it easier for store owners who might wish to translate, or otherwise mod
 | `%%LNG_CouponCode%%` | "Coupon Code" |
 | `%%LNG_CouponCodeIntro%%` | "If you have a coupon code, enter it in the box below and click 'Go'." |
 | `%%LNG_CreateAccount%%` | "Create Account" |
-| `%%LNG_CreateAccountIntro1%%` | <b>Checkout:</b> "Fill in the fields below to create a "<br><b>Express Checkout:</b> "Checkout faster" |
-| `%%LNG_CreateAccountIntro2%%` | <b>Checkout:</b> "account. You'll be able to shop and check out faster on your next visit, check your order status online and optionally receive updates on special offers and events."<br><b>Express Checkout:</b> "Save multiple addresses in your address book" |
+| `%%LNG_CreateAccountIntro1%%` | **Checkout:** "Fill in the fields below to create a "<br>**Express Checkout:** "Checkout faster" |
+| `%%LNG_CreateAccountIntro2%%` | **Checkout:** "account. You'll be able to shop and check out faster on your next visit, check your order status online and optionally receive updates on special offers and events."<br>**Express Checkout:** "Save multiple addresses in your address book" |
```
Comment (slsriehl) [inline docs/legacy/blueprint-themes/lng-variables.md:210]:
```suggestion
| `%%LNG_CreateAccountIntro2%%` | **Checkout:** "account. You'll be able to shop and check out faster on your next visit, check your order status online, and optionally receive updates on special offers and events."<br>**Express Checkout:** "Save multiple addresses in your address book" |

```

**Review by slsriehl — CHANGES_REQUESTED** [review]
@bc-andreadao Let's consult PSEs to make a call on whether the tables of templating examples in `lng` and `email-templates` are supposed to represent string literals (display the actual html tags the merchant should code) or formatted text. 

- If they should be string literals, revert md bold back to HTML tags and find/replace html tag characters with html-encoded ones so that they display.
- Otherwise, convert all the `<b>` tags to `**` per this ticket

cc. @robincoferbc @bc-mikToscano thoughts? 
- how do merchants/partners use this? 
- are they customizing strings with these template variables, or primarily consulting the tables to see what some preformatted drop-in snippets using the tags look like?
- how many people actually use the blueprint email templates vs stencil?

**Comment by bc-mikToscano** [comment]
Hey @slsriehl 
> * how do merchants/partners use this?
> * are they customizing strings with these template variables, or primarily consulting the tables to see what some preformatted drop-in snippets using the tags look like?

Hard to give a definitive, but from experience it would be the former. Merchants/Partners using blueprint will usually referencing the table with customisation already in mind, and would find themselves directed to this page to clarify the exact text that the variables call; In that they would be copying the 'variable' context to use in their templates, not the 'default string value'. 
If the 'Default String Value' column is designed for reference intent, I see no issues being raised from updating the formatting from html to md

> * how many people actually use the blueprint email templates vs stencil?
Another hard to answer; support continue to push Stencil as the preferred templates, however the number of ENT Merchants & Partners who support them are still enough to warrant ongoing access to this documentation for the foreseeable future

Hope this helps. Happy to give further clarification if needed

**Comment by slsriehl** [comment]
Many thanks, @bc-mikToscano.

@bc-andreadao Let's go ahead and convert all these examples to literal strings of HTML, so that readers can see the markup characters and templating variables in action. Most of the character reference you need is available in [this HTML 8859 reference](https://www.w3schools.com/charsets/ref_html_8859.asp). The "Entity Name" column is the most readable encoding option. I can give guidance on this if you're unsure what I mean. Give things a preview in Stoplight to make sure stuff is parsing.

**Comment by bc-andreadao** [comment]
@slsriehl The preview looks fine (has literal string templates), but the check failed. Should we override the check?

**Comment by slsriehl** [comment]
@bc-andreadao I'll review this evening with tonight's PRs. changing labels...

**Review by slsriehl — APPROVED** [review]
magnificent perseverance.

**Pull request merged**